src/Microsoft.VisualStudio.LanguageServices.Razor/DefaultTagHelperResolver.cs

```diff
@@ -43,7 +40,7 @@ public DefaultTagHelperResolver(ErrorReporter errorReporter, Workspace workspace
                 // when it's disconnected (user stops the process).
                 //
                 // This will change in the future to an easier to consume API but for VS RTM this is what we have.
```

Comment:
These comments are a bit out of date. Can we can update this to make it seem less temporary? I think we'll have this complexity here forever.

Reply:
Will do